experimental/dds/tree/src/LogViewer.ts

```diff
@@ -130,8 +130,8 @@ export class CachingLogViewer implements LogViewer {
 			// This avoids having to invalidate cache entries when concurrent edits cause local revision
 			// numbers to change when acknowledged.
 			if (i < this.log.numberOfSequencedEdits) {
-				const revision = i + 1; // Revision is the result of the edit being applied.
-				this.sequencedSnapshotCache.set(revision, currentSnapshot);
+				// Revision is the result of the edit being applied.
+				this.sequencedSnapshotCache.set(/* revision: */ i + 1, currentSnapshot);
```

Comment:
What linter rule motivates this change?

Reply:
'no-shadow'.  The local 'revision' variable shadows the 'revision' argument passed to 'getSnapshot()'.

(In the 'Files Changed' view, you can push the up arrow to see the preceding lines.)